### DISCUSSION TITLE
Future Release

### DISCUSSION THREAD

**Discussion by AndrewJackson2020** [pgbouncer/pgbouncer#1089] [General]
Is there any idea when the next release will be cut? 

I've noticed 2 issues related to passthrough auth: 
1. passthrough auth not pulling passwords from the database when setting user level connection limits in the pgbouncer.conf
2. I pgbouncer crashing when a user that is being pulled in via passthrough has connection limits added to pgbouncer.conf.

These issues impact the most recent release but when I build the binary from source it seems to have been fixed  at some point after the last release. Would love to be able to use a pgbouncer release that have the above 2 issues fixed.

**Comment by JelteF** [comment]
No promises, but my current goal is somewhere this summer.

**Reply by ronalders** [reply]
Meteorological summer is officially over :-) Any chance the next PgBouncer release made it onto the autumn roadmap? 🍂

I’m especially looking forward to getting #1555 in a release — the fix is already happily sitting in master, just waiting to escape into the wild 😄.

Any updated ETA? Thanks!

**Reply by AndrewJackson2020** [reply]
There is one queued up for this week.

https://github.com/pgbouncer/pgbouncer/pull/1615

**Reply by ronalders** [reply]
Awesome! 🎉 That’s exactly the update I was hoping for. Looking forward to the release
